### PR TITLE
sc 154171/update Roku SDK hello app readme and comments

### PR DESCRIPTION
**Pull request by ember-stevens**
[SC-154171](https://app.shortcut.com/launchdarkly/story/154171/update-roku-sdk-hello-app-readme-and-comments-to-standard-design): After receiving some feedback that it would be helpful if the Hello apps were more standardized, Docs team is working on updating them to [match the spec](https://launchdarkly.atlassian.net/wiki/spaces/ENG/pages/1499562073/SDK+hello+world).

I could not test this locally, so there may be issues I didn't catch.

### CODE REVIEW

File: app/components/AppScene.brs [hunk 1]
```diff
@@ -18,14 +24,18 @@ end function
...
-    user = LaunchDarklyUser("user-key")
+  
+    // Set up the user properties. This user should appear on your LaunchDarkly users dashboard
+    // soon after you run the demo.
+    user = LaunchDarklyUser("example-user-key")
+    user.setName("Sandy")
 
     LaunchDarklySGInit(config, user)
-
+    // TKTK initialization success/failure messages needed.
```
Comment (ember-stevens) [inline app/components/AppScene.brs:37]: Ideally, we'd add initialization success/failure messages here per step 3: https://launchdarkly.atlassian.net/wiki/spaces/ENG/pages/1499562073/SDK+hello+world#Example-README.md

**Review by louis-launchdarkly — APPROVED** [review]
The change looks good to me.

File: README.md [hunk 2]
```diff
@@ -1,7 +1,18 @@
...
-We've built a simple application that demonstrates how LaunchDarkly's SDK works. Below, you'll find the basic build procedure, but for more comprehensive instructions, you can visit your [Quickstart page](https://app.launchdarkly.com/quickstart#/).
+We've built a simple application that demonstrates how LaunchDarkly's SDK works. 
+
+Below, you'll find the basic build procedure, but for more comprehensive instructions, you can visit your [Quickstart page](https://app.launchdarkly.com/quickstart#/) or the [Roku reference guide](https://docs.launchdarkly.com/sdk/client-side/roku).
 
 ## Build instructions
-1. Copy your Mobile SDK key and feature flag key from your LaunchDarkly dashboard into `app/components/AppScene.brs`
-2. Run `make`
-3. Upload `app.zip` to your device
+1. Edit `app/components/AppScene.brs` and set the value of `mobileKey` to your LaunchDarkly mobile SDK key. If there is an existing boolean feature flag in your LaunchDarkly project that you want to evaluate, set `featureFlagKey` to to the flag key.
```
Comment (mmrj) [inline README.md:7]:
```suggestion
1. Edit `app/components/AppScene.brs` and set the value of `mobileKey` to your LaunchDarkly mobile SDK key. If there is an existing boolean feature flag in your LaunchDarkly project that you want to evaluate, set `featureFlagKey` to the flag key.
```

File: README.md [hunk 3]
```diff
@@ -1,7 +1,18 @@
...
+```brightscript
+featureFlagKey = "my-boolean-flag"
+
+mobileKey = "1234567890abcdef"
+```
+
+2. On the command line, run `make`.
+3. Upload `app.zip` to your device.
+
+You should see the message "Feature flag '<flag key>' is <true/false> for this user".
```
Comment (mmrj) [inline README.md:18]:
```suggestion
You should receive the message "Feature flag '<flag key>' is <true/false> for this user".
```
per your suggestion earlier

**Review by mmrj — CHANGES_REQUESTED** [review]
Thanks Ember! 

Requesting changes so that this doesn't get merged with the "TKTK" still in `AppScene.brs`, but it looks good to me.

**Review by mmrj — APPROVED** [review]

File: app/components/AppScene.brs (hunk 1 above)
Comment (eli-darkly) [reply app/components/AppScene.brs]: I'm the closest thing we have. I don't know off the top of my head how (or whether) this works in Roku, but I'll take a look.

**Pull request merged**